vhdbuilder/packer/components.json

```diff
@@ -2,351 +2,425 @@
   "ContainerImages": [
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/kubernetes-dashboard:*",
```

Comment:
Did you manually (by checking with `aks-rp` code) determine which versions of components were not being used by rp? Or was there another approach you had used?
I'm trying to understand it so that we can perhaps replicate this process for other versions. Thanks.